apps/src/netsim/NetSimVisualization.js

```diff
@@ -322,7 +303,6 @@ NetSimVisualization.prototype.onNodeTableChange_ = function (rows) {
   // Update collection of VizNodes from source data
   this.updateVizEntitiesOfType_(NetSimVizSimulationNode, tableNodes, function (node) {
     var newVizNode = new NetSimVizSimulationNode(node);
-    newVizNode.setDnsMode(this.netsim_.getDnsMode());
```

Comment:
I'm not sure this should be removed.  Without this code, a new node doesn't get a DNS mode until a DNS mode change event occurs, which can lead to situations like this:

![screenshot from 2015-07-28 12 46 08](https://cloud.githubusercontent.com/assets/1615761/8941814/cba5a9ac-3526-11e5-88ee-8307648cc061.png)

DNS Mode: None, but a node has a hidden address.

I'll see if I can come up with a test that catches this case.